packages/replay-next/src/contexts/FocusContext.tsx

```diff
@@ -133,27 +140,16 @@ export function FocusContextRoot({ children }: PropsWithChildren<{}>) {
       }
 
       setBias(bias);
-      setRange(range);
-
-      // Focus values may change rapidly (e.g. during a click-and-drag)
-      // In this case, React's default high/low priority split is helpful, but we can do more.
-      // Debouncing a little before starting the transition helps avoid sending a lot of unused requests to the backend.
-      if (debounce) {
-        debouncedSetDeferredRange(range);
-      } else {
-        startTransition(() => {
-          setDeferredRange(range);
-        });
-      }
+      setRangeForDisplay(range);
```

Comment:
This will trigger the effect above which updates `range` and `rangeForSuspense`.